packages/web/src/builders/package/package.impl.ts

```diff
@@ -98,12 +101,16 @@ export function run(
             }),
             postcss({
               inject: true,
-              extract: false,
+              extract: options.extractCss,
               autoModules: true,
               plugins: [autoprefixer]
             }),
             localResolve(),
-            resolve({ extensions: fileExtensions }),
+            resolve({
+              mainFields: ['module', 'es2015', 'main'],
+              preferBuiltins: true,
```

Comment:
Turns warning off by default if user imports a built-in module (e.g. `path`). This is common if the React library is also meant for SSR (so `stream` and `path` are likely used). We never want to use a locally resolved version of those modules.

Reply:
Yeah, `styled-components` uses `stream` from node for the SSR portion of the package. THis will hide those warnings, and users should not expect Node native modules nor local alternatives to be bundled anyway.